### PR TITLE
Power model add

### PR DESCRIPTION
**Pull request by Tess-LaCoil**
Added the power law model.

### CODE REVIEW

**Review by fontikar — APPROVED** [review]
Looks great to me! Nice work so glad we have a workflow now! :D Have no viewed Stan files in much detail, @dfalster can we lean on you for that please? 🙏

**Review by dfalster — CHANGES_REQUESTED** [review]
Nice work Tess. Looks great. However, I suggest we write power law as  dx/dt = a (x/x_c)^b. Your  implementation has x_c=1. a general guide would be that x_c ~= mean(x), but rather than estimating from the data, it would be better to set this in the model setup as a specified quantify. Without this, the slope and intercept become heavily correlated and hard to converge.

**Comment by Tess-LaCoil** [comment]
Changes to power law model done. @dfalster I don't know if the data generation process is sane as I produce y_bar values and pass them along with the other DE parameters to used to determine true sizes over time. Those true values also get passed to the model, but I assume in practice these would be estimated from the data or something.

File: inst/stan/power_multi_ind.stan [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+// Data structure
+data {
+  real step_size;
+  int n_obs;
+  int n_ind;
+  real y_obs[n_obs];
+  int obs_index[n_obs];
+  real time[n_obs];
+  int ind_id[n_obs];
+  real y_bar[n_ind];
```
Comment (dfalster) [inline inst/stan/power_multi_ind.stan:60]: Sorry if this wasn't clear. This should no vary by individuals, it's a single value applying to the whole dataset

File: inst/stan/power_multi_ind.stan [hunk 2]
```diff
@@ -0,0 +1,149 @@
+// Power law model with multiple individuals
+
+functions{
+  //DE function for use with Runge-Kutta method
+  //pars = [ind_coeff, ind_power]
+  real DE(real y, real y_bar, vector pars){
```
Comment (dfalster) [inline inst/stan/power_multi_ind.stan:6]: why not include y_bar in pars?

**Review by dfalster — CHANGES_REQUESTED** [review]
Sorry , more changes!

File: inst/stan/power_multi_ind.stan (hunk 2 above)
Comment (Tess-LaCoil) [reply inst/stan/power_multi_ind.stan:6]: Can do.

**Review by dfalster — APPROVED** [review]

**Pull request merged**